src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/JsonMetadataServices.cs

```diff
@@ -48,7 +48,7 @@ public static JsonPropertyInfo CreatePropertyInfo<T>(JsonSerializerOptions optio
                 throw new InvalidOperationException(SR.Format(SR.FieldCannotBeVirtual, nameof(propertyInfo.IsProperty), nameof(propertyInfo.IsVirtual)));
             }
 
-            return new JsonPropertyInfo<T>(options, propertyInfo);
+            return new JsonPropertyInfo<T>(propertyInfo, options);
```

Comment:
any particular reason for changing order or params?

Reply:
Regular convention is passing `JsonSerializerOptions` as the last parameter, so I just made that method signature conform with it.